config/environment.js

```diff
@@ -297,6 +298,10 @@ module.exports = function (environment) {
       ENV.billingEndpoint = BILLING_ENDPOINT;
     }
 
+    if (GITHUB_ORGS_OAUTH_ACCESS_SETTINGS_URL) {
+      ENV.providers.github.paths.accessSettings = GITHUB_ORGS_OAUTH_ACCESS_SETTINGS_URL;
+    }
```

Comment:
Wouldn't it be better to set the value in place, where it's defined, instead of suddenly updating it somewhere else? Just thinking that it might be unexpected when debugging. All env vars are available in providers' config as well. Also this would allow to keep the `deepFreeze`. What do you think?